src/main/resources/templates/c/test/Converter.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+import sys, getopt
+from xml.sax.saxutils import escape
+
+def usage():
+    print('python3 <script> -i <inputfile> -o <outputfile>')
+
+def main(argv):
+    inputfile = ''
```

Comment:
To keep it consistent, when do you use `'` and when `"`?
Besides that, I personally do not like untyped python. This might not be a problem if you are writing only small scrips like this one, but for larger frameworks it is important in my eyes to add types in python.

Reply:
Usually `"` for natural language, and `'` for symbolic strings. But I agree, I used it inconsistently, will be fixed. Good point about the typing, but I would like to omit it here, as it's a really small and simple helper script.

Reply:
Yes, we can omit it here since it's just a small script.